pages/glossary/distinguishable-colors.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Distinguishable colors
+key: distinguishable-colors
+unambiguous: true
+objective: true
+---
+
+Two elements have _distinguishable colors_ if the [highest possible contrast][] between the [foreground colors][] of the [text nodes][] of the two elements is at least 3.0:1, or if the [highest possible contrast][] between the [background colors][] of the two elements is at least 3.0:1.
```

Comment:
Foreground colors include aliased pixels. Even if the color property is the same, the highesst possible contrast between foreground colors will likely always be 3:1 for text with aliasing.

Reply:
I don't think we should create a definition of "foreground colors without anti-aliased pixels". Would this solution work for you?